models/mixpanel.yml

```
@@ -296,6 +296,9 @@ models:
 
       - name: event_frequencies
         description: JSON of the frequency of each `event_type` during this user session.
+      
+      - name: user_id
+        dev_snowflake: Coalescing of `device_id` and `people_id`.
```

What is the `dev_snowflake` config? I assume it should be `description`?
```suggestion
        description: Coalescing of `device_id` and `people_id`.
```